databases/week1/Lucy_Chyzhova_HW.sql

```diff
@@ -0,0 +1,52 @@
+-- Find out how many tasks are in the task table         
+SELECT COUNT(id) AS amount_of_task  -- or (*) 
+FROM task; 
+
+-- Find out how many tasks in the task table do not have a valid due date  
+SELECT COUNT(task.id) AS task_count 
+FROM task 
+WHERE due_date is null; 
+
+-- Find all the tasks that are marked as done            
+SELECT task.title, status.name AS status_as_text
+FROM task 
+INNER JOIN status ON status.id = task.status_id
+WHERE status.name like 'done';
+
+-- Find all the tasks that are not marked as done         
+SELECT task.title, status.name AS status_as_text
+FROM task 
+INNER JOIN status ON status.id = task.status_id
+WHERE status.name NOT like 'done';
```

Comment:
Really nice! :star:
Most of the time these two are solved by just using the task.status_id in the WHERE clause but this solution is even better.

One small thing I would mention is that when matching on exact strings it is usually better to use the equals operator instead of LIKE `WHERE status.name = 'done';` for example